cdap-ml-ext-vertexai/src/main/java/io/cdap/cdap/ml/ext/vertexai/VertexAIConfiguration.java

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
Follow this hierarchy for class definition:

public class X {
    // public constants
    // private constants

   // public variables
   // private variables

   // Default Constructor

   // Parametrized Constructor

   // Public Setters / Getters 

   // Public Functions

   // Private Functions

   // Nested Class with the same hierarchy

}

Reply:
Fix t/o